packages/airlock_digital/data_stream/execution_histories/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,383 @@
+  - remove:
+      field:
+        - organization
+        - division
+        - team
+      ignore_missing: true
+      if: ctx.organization instanceof String && ctx.division instanceof String && ctx.team instanceof String
+      tag: remove_agentless_tags
+      description: >-
+        Removes the fields added by Agentless as metadata, as they can collide with ECS fields.
+  - json:
+      field: event.original
+      tag: json_event_original
+      target_field: json
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - append:
+      field: event.type
+      tag: append_info_into_event_type
+      value: info
+  - append:
+      field: event.category
+      tag: append_host_into_event_category
+      value: host
+  - set:
+      field: event.kind
+      tag: set_event_kind
+      value: event
+  - rename:
+      field: json.checkpoint
+      tag: rename_checkpoint
+      target_field: airlock_digital.execution_histories.checkpoint
+      ignore_missing: true
+  - set:
+      field: event.id
+      tag: set_event_id_from_execution_histories_checkpoint
+      copy_from: airlock_digital.execution_histories.checkpoint
+      ignore_empty_value: true
+  - rename:
+      field: json.commandline
+      tag: rename_commandline
+      target_field: airlock_digital.execution_histories.commandline
+      ignore_missing: true
+  - set:
+      field: process.command_line
+      tag: set_process_command_line_from_execution_histories_commandline
+      copy_from: airlock_digital.execution_histories.commandline
+      ignore_empty_value: true
+  - date:
+      field: json.datetime
+      tag: date_datetime
+      target_field: airlock_digital.execution_histories.datetime
+      formats:
+        - ISO8601
+      if: ctx.json?.datetime != null && ctx.json.datetime != ''
+      on_failure:
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - set:
+      field: '@timestamp'
+      tag: set_@timestamp_from_execution_histories_datetime
+      copy_from: airlock_digital.execution_histories.datetime
+      ignore_empty_value: true
+  - rename:
+      field: json.filename
+      tag: rename_filename
+      target_field: airlock_digital.execution_histories.filename
+      ignore_missing: true
+  - set:
+      field: file.path
+      tag: set_file_path_from_execution_histories_filename
+      copy_from: airlock_digital.execution_histories.filename
+      ignore_empty_value: true
+  - gsub:
+      field: file.path
+      pattern: "\\\\"
+      replacement: "/"
+      ignore_failure: true
```

```suggestion
      pattern: '\\'
      replacement: '/'
      ignore_failure: true
```
but is this `\` → `/` transform intentional?

---

Yes, the transformation is intentional—it’s done to build the file path.